src/services/ResourceManager.cxx

```diff
@@ -28,6 +49,116 @@ ResourceManager::ResourceManager(GameService::ServiceTuple &services) : GameServ
 #endif // USE_AUDIO
 }
 
+
+//#ifdef USE_AUDIO
+static int LoadAudioWithOggVorbis(std::string path, DecodedAudioData& dAudioBuffer)
+{
+  //0 means success
+  //-1 means failure
```

Comment:
Please document return types in doxygen in the header instead of in thx cxx file as normal comment